entries/s/sellix.io.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Sellix": {
+    "domain": "sellix.io",
+    "tfa": [
+      "email",
+      "totp"
+    ],
+    "documentation": "https://help.sellix.io/en/articles/4502811-how-do-i-enable-multi-factor-authentication",
+    "categories": [
+      "retail",
+      "payments",
+      "cryptocurrencies"
```

Comment:
I'm not sure all these categories fit sellix.io. Could you describe a bit more what the service does? Maybe comparing it to other companies in the space?

Currently it's labeled as a retail company (i.e. Target,Nordstrom), Payment provider (i.e. PayPal, Klarna, Stripe) and cryptocurrency company (a la Coinbase, Crypto.com)

Reply:
@Carlgo11 It's an e-commerce service. It allows for the sale of physical or digital goods. I also feel that it is a payment provider since it accepts Cryptocurrency payments on its own, no use of a 3rd party unlike with the other payment methods such as Credit Card.

Reply:
Sounds like a mix between Shopify and Stripe then. Remove the retail category and it's all good.